pkgs/applications/misc/kjv/default.nix

```diff
@@ -1,43 +1,41 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, readline }:
 
 let
+  patchPrefix = "https://github.com/samuelgrf/kjv/commit/";
 
-patch-base = "https://github.com/LukeSmithxyz/kjv/commit/";
-
-add-apocrypha = fetchpatch {
-  url = patch-base + "b92b7622285d10464f9274f11e740bef90705bbc.patch";
-  sha256 = "0n4sj8p9m10fcair4msc129jxkkx5whqzhjbr5k4lfgp6nb1zk8k";
-};
-
-add-install-target = fetchpatch {
-  url = patch-base + "f4ad73539eb73f1890f4b791d8d38dd95900a4a4.patch";
-  sha256 = "1yzj72i5fkzn2i4wl09q6jx7nwn2h4jwm49fc23nxfwchzar9m1q";
-};
+  add-apocrypha = fetchpatch {
+    url = patchPrefix + "3f5cc5cbb7b5a94d5dd8b17a450fac4a3e162a32.patch";
+    sha256 = "sha256-5xxeem3Fg1C0L+zp7Ks/T+3dSXLjpXcM5morLFQYk74=";
+  };
 
+  add-install-target = fetchpatch {
+    url = patchPrefix + "87cef02980e9c3450855f8f3478d354bdf7d5960.patch";
+    sha256 = "sha256-+3uCP609+VRTDI3+PN6V/CV2APDGbqNMIrLJY1T1aS8=";
+  };
 in
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "kjv";
-  version = "unstable-2018-12-25";
+  version = "unstable-2021-03-11";
 
   src = fetchFromGitHub {
     owner = "bontibon";
-    repo = pname;
-    rev = "fda81a610e4be0e7c5cf242de655868762dda1d4";
-    sha256 = "1favfcjvd3pzz1ywwv3pbbxdg7v37s8vplgsz8ag016xqf5ykqqf";
+    repo = "kjv";
+    rev = "108595dcbb9bb12d40e0309f029b6fb3ccd81309";
+    hash = "sha256-Z6myd9Xn23pYizG+IZVDrP988pYU06QIcpqXtWTcPiw=";
   };
 
   patches = [ add-apocrypha add-install-target ];
 
-  makeFlags = [
-    "PREFIX=${placeholder "out"}"
-  ];
+  buildInputs = [ readline ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
 
   meta = with lib; {
     description = "The Bible, King James Version";
     homepage = "https://github.com/bontibon/kjv";
-    license = licenses.publicDomain;
-    maintainers = [ maintainers.jtobin ];
+    license = licenses.unlicense;
```

Comment:
Should this still be `publicDomain`?  Not sure what the differences is between that and `unlicense`.

Reply:
AFAIK the difference is that the Unlicense specifies limitations on liability and warranty.
I made this change because the [LICENSE file in kjv's repo](https://github.com/bontibon/kjv/blob/108595dcbb9bb12d40e0309f029b6fb3ccd81309/LICENSE) contains the Unlicense.

I am not a lawyer and this is not legal advice.